plutus-metatheory/src/Algorithmic/CEK.lagda.md

```diff
@@ -353,16 +355,16 @@ step (s ; ρ ▻ Λ L) = s ◅ V-Λ L ρ
 step (s ; ρ ▻ (L ·⋆ A / refl)) = (s , -·⋆ A) ; ρ ▻ L
 step (s ; ρ ▻ wrap A B L) = (s , wrap-) ; ρ ▻ L
 step (s ; ρ ▻ unwrap L refl) = (s , unwrap-) ; ρ ▻ L
-step (s ; ρ ▻ con c) = s ◅ V-con c
+step (s ; ρ ▻ con c refl) = s ◅ V-con c
```

Comment:
could you use a pattern synonym here?

Reply:
I could. The synonym cannot be named `con` (could be named something like `const`) even if we rename the real `con` constructor to `con_/_` because it clashes with the `con` constructor for types.